syntax = "proto3";

import "messages/script_submit.proto";
import "messages/synchronous_reply.proto";

package internal;

service ApiToCore {
    rpc ScriptSubmit (ScriptSubmitRequest) returns (SynchronousReply);
}